app/views/users/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<h2><%= @user.name %>'s Dashboard</h2>
+
+<%= button_to "Discover Movies", "/users/#{@user.id}/discover", method: :get %>
+
+<section id="viewing_parties">
```

Comment:
love the use of section for testing and within blocks. I'll make sure to implement that in my own moving forward